packages/backend/eslint.config.js

```diff
@@ -2,7 +2,8 @@ export default [
   {
     rules: {
       semi: "error",
-      "prefer-const": "error"
+      "prefer-const": "error",
+      "react/no-unescaped-entities": 0 
```

Comment:
nice! thanks for adjusting the rules so the linter can stop nagging us about our quotes/ spacing